src/router/console/acls.js

```diff
@@ -25,7 +25,8 @@ export default [
           title: i18n.t('UserLoginACLs'),
           app: 'acls',
           resource: 'loginacl',
-          disableOrgsChange: true
+          disableOrgsChange: true,
+          licenseRequired: true
         },
         children: [
           {
```

Comment:
There is no clear difference between your code snippets that could indicate any inconsistencies, issues, or inefficiencies related to the import statement or the use of variables. It seems like there might be an oversight; however, without more context on what "export" and its arguments represent within the provided lines, it's difficult to analyze whether these statements contain irregularities or need specific improvement(s).